i18n/en.json

```diff
@@ -2352,7 +2352,7 @@
   },
   {
     "id": "app.cluster.404.app_error",
-    "translation": ""
+    "translation": "Cluster API: Endpoint not found."
```

Comment:
Which endpoint isn't found?

Reply:
Here I didn't know what to put exactly there. Basically, it is the 404 error for the cluster related endpoints (used to sync nodes). It wont happen to the users, or at least it mustn't.